fdbserver/VersionedBTree.actor.cpp

```diff
@@ -5255,6 +5261,7 @@ class VersionedBTree {
 				    .detail("InstanceName", self->m_pager->getName())
 				    .detail("UsingEncodingType", self->m_encodingType)
 				    .detail("ExistingEncodingType", self->m_header.encodingType);
+				throw unexpected_encoding_type();
```

Comment:
nit: was this missing from initial commit? Am hoping the callers are able to handle this error.

Reply:
Previously redwood only fail with `unexpected_encoding_type` after it read a page with unexpected encoding. Here I'm failing it early on `init`.